### ISSUE TITLE
Composer install fails on fabiang/sasl: dev-rspauth

### ISSUE THREAD

**Issue by richp10** [kadet1090/nucleus#1]
Trying to install nucleus using composer, install fails because it cannot find fabiang/sasl: dev-rspauth

Looking at the fabiang archive, there does not seem to be a dev-rspauth tag..

(Project looks great and I'm happy to provide any other feedback if I can get installed!)

**Comment by kadet1090** [comment]
Yes, i know about that issue. However, this is more related to composer. fabiang/SASL has one bug (or maybe it's intentional behaviour) and I had to fork it, and work with that. And I'm still waiting for response on my PR. Sadly, composer is not looking into repositories defined in dependencies and throws error - which is quite stupid. There is one workaround. You can define additional repository in your `composer.json`:

``` json
"repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/kadet1090/sasl"
    }
  ],
```

 or wait little bit (probably one day) I will try to do something to not to have to maintain my own fork of sasl with added 3 lines of code.

**Comment by kadet1090** [comment]
I've pushed changes, you can checkout `dev-master` or `dev-fix-sasl` version which is based on 0.2 and will become 0.2.1 if everything will be okey. Please notify me if everything work, thx.

**Comment by richp10** [comment]
Thanks - that worked
